dev/fattest.simplicity/bnd.bnd

```diff
@@ -39,6 +39,7 @@ Export-Package: \
 	componenttest.rules;version=1.0.16,\
 	componenttest.rules.repeater;version=1.0.16,\
 	componenttest.topology.database;version=1.0.16,\
+	componenttest.topology.database.container;version=1.0.16,\
```

Comment:
Can this be put under `componenttest.containers.database` instead?  that way it shows up under the other container focused package.

Reply:
I just did alphabetical order to match the rest of the exports as to order.